docs/endpoints/post-token-refresh.md

```diff
@@ -42,9 +43,11 @@ Using either of the following parameters in a [POST /token/generate](post-token-
 
 For details and Python script examples, see [Encrypting Requests and Decrypting Responses (Python script example)](../getting-started/gs-encryption-decryption#uid2_requestpy).
 
+{**GWH/SW_04. I would dearly love to have a plain request example AS WELL AS this link to a Python script. I feel strongly that we shoudl have an example request and response in the doc for each operation. Is it possible I could get one? Fictitious of course. user@example.com would be fine.**}
```

Comment:
Are you saying we should have a uid2_request.py example on each page of token/refresh and token/generate instead of just having 1 combined script on https://unifiedid.com/docs/getting-started/gs-encryption-decryption page?

Reply:
@jon8787 please give any advice if you have any

Reply:
token/generate already has an example.

For token/refresh, you could copy the refresh example from the script on that page:

> python3 uid2_request.py https://prod.uidapi.com/v2/token/refresh --refresh-token AAAAAxxJ...(truncated, total 388 chars) v2ixfQv8eaYNBpDsk5ktJ1yT4445eT47iKC66YJfb1s=

I suspect it's not very helpful since the user won't know what the refresh-token and refresh_response_key should be.

Reply:
Sunny: no just saying a plain example with no script. If that' possible. Jonathan: no prob will not change it. I was thinking maybe a cURL example or something. Something really simple.